receiver/jaegerreceiver/jaegerreceiverdeprecated/metadata.yaml

```diff
@@ -0,0 +1,17 @@
+type: jaeger
```

Comment:
don't think you'll need the metadata files in the deprecated components that aren't modules